keepassxc-browser/content/ui.js

```diff
@@ -143,8 +143,8 @@ kpxcUI.setIconPosition = function(icon, field, rtl = false, segmented = false) {
         top = iconOffset[1];
     }
 
-    const scrollTop = document.scrollingElement ? document.scrollingElement.scrollTop : 0;
-    const scrollLeft = document.scrollingElement ? document.scrollingElement.scrollLeft : 0;
+    const scrollTop = document.defaultView ? document.defaultView.scrollY : document.scrollingElement.scrollTop;
+    const scrollLeft = document.defaultView ? document.defaultView.scrollX : document.scrollingElement.scrollLeft;
```

Comment:
Could also do something like this:

```suggestion
    const scrollLeft = document.defaultView?.scrollX ?? document.scrollingElement?.scrollLeft ?? 0;
```

Reply:
Gotta bump up the minimum Firefox version for these (67 -> 74 so no biggie). But those are a nice cleanup.

Reply:
I think both `?.` and `??` are in use in small parts of the code already. I think I added the `?.` before I even considered compatibility.

Reply:
If you already use browser greater than the minimum version there's no problems. But if you use e.g. Firefox 67 I bet those will not work and will throw an exception. A rare situation of course, because I doubt no one is using versions that old.